src/services/user/services/AdminUsers.js

```diff
@@ -145,12 +145,18 @@ class AdminUsers {
 			});
 
 			// sorting by class and by consent is implemented manually,
-			// as classes and consents are fetched from seperate db collection
-			const classSortParam = (($sort || {}).class || {}).toString();
+			// as classes and consents are fetched from separate db collection
+			const classSortParam = (($sort || {}).classes || {}).toString();
 			if (classSortParam === '1') {
-				users.sort((a, b) => (a.classes[0] || '').toLowerCase() > (b.classes[0] || '').toLowerCase());
+				users.sort((a, b) => (a.classes[0] || '').toLowerCase().localeCompare((b.classes[0] || '').toLowerCase(), undefined, {
+					numeric: true,
+					sensitivity: 'base',
+				}));
 			} else if (classSortParam === '-1') {
-				users.sort((a, b) => (a.classes[0] || '').toLowerCase() < (b.classes[0] || '').toLowerCase());
+				users.sort((a, b) => (b.classes[0] || '').toLowerCase().localeCompare((a.classes[0] || '').toLowerCase(), undefined, {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 134. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5502439)